static/src/javascripts/projects/common/modules/ui/sticky-social.js

```diff
@@ -78,18 +78,10 @@ define([
 
     function init() {
         var testVariant = ab.getTestVariant('ShareButtons2'),
-            referrer,
             socialContext;
 
         if (testVariant.indexOf('referrer') > -1) {
-            referrer = ((window.location.hash + '').match(/referrer=([^&]+)/) || [])[1] || document.referrer || '',
-
-            socialContext = [
-                {id: 'facebook', matchReferrer: 'facebook.com', isApp: detect.isFacebookApp},
-                {id: 'twitter', matchReferrer: 't.co', isApp: detect.isTwitterApp}
-            ].filter(function (social) {
-                return referrer.indexOf(social.matchReferrer) > -1 || social.isApp();
-            })[0];
+            socialContext = detect.socialContext();
```

Comment:
Below this - on line 95 - you need `socialContext` instead of `socialContext.id`